trot_gait_node:
  ros__parameters:
    use_imu: False
    gait_period: 0.5
    gait_height: 0.03
    ground_penetration: 0.005
    # fixed_forward_body: 0.02
    # resolution_first_fraction: 0.33
    # period_first_fraction: 0.66
